### DISCUSSION TITLE
graphscope是否有适配centos的包可用

### DISCUSSION THREAD

**Discussion by timer-2117** [alibaba/GraphScope#3428] [Q&A] [answered]
如题，最近在centos上希望跑ldbc测试，deb包暂时无法满足需求，所以想请问一下是否有rpm包之类的可以提供

**Comment by luoxiaojian** [comment, marked as answer]
你好，因为我们的测试、部署主要都还是在ubuntu上，centos上没有充分测试过，如果要提供rpm包，可能要等充分测试之后
建议可以从源码编译、测试，如果遇到什么问题，我们可以帮你解决
